### PR TITLE
Features/react router

### PR DESCRIPTION
**Pull request by mikolajgraczyk**
React router working on header buttons switching between popular movies and popular people page.

### CODE REVIEW

**Review by HelkaPL — APPROVED** [review]

**Comment by marcin10lw** [comment]
How about we use react-router v6? It's easier, has more features and will simplify our work with search parameters.

**Comment by mikolajgraczyk** [comment]
I'll work on it today

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
+import ActorTile from "./features/actors/ActorTile";
 import Header from "./common/Header";
-import { HashRouter } from "react-router-dom";
+import { Routes, Route } from "react-router-dom";
+
 
 function App() {
   return (
-    <HashRouter>
+    <>
```
Comment (marcin10lw) [inline src/App.js:11]: We need to wrap the App in HashRouter

File: src/App.js [hunk 2]
```diff
@@ -1,22 +1,33 @@
...
       <Header />
-      <Section
-        title="Popular movies"
-        body={
-          <MovieList />
-        }
-      />   
+      <Routes>
```
Comment (marcin10lw) [inline src/App.js:13]: When we open our page we should be redirected to movies page.
To do that we can make a route `<Route path="/" element={<Navigate to="/movies" />} />` 😉

File: src/common/Header/index.js [hunk 3]
```diff
@@ -28,8 +29,12 @@ const Header = () => {
                         <TitleText>Movies Browser</TitleText>
                     </TitleWrapper>
                     <ButtonsWrapper>
-                        <HeaderButton>MOVIES</HeaderButton>
-                        <HeaderButton>PEOPLE</HeaderButton>
+                        <StyledNavLink to="/movies">
```
Comment (marcin10lw) [inline src/common/Header/index.js:32]: I think we dont need buttons for that.
How about we just use `<StyledNavLink to="/movies" /> ` with out button styles?

File: src/common/Header/index.js [hunk 4]
```diff
@@ -28,8 +29,12 @@ const Header = () => {
...
+                            <HeaderButton>MOVIES</HeaderButton>
+                        </StyledNavLink>
+                        <StyledNavLink to="/people">
```
Comment (marcin10lw) [inline src/common/Header/index.js:35]: Same thing with StyledNavLink here 😀

File: src/common/Header/styled.js [hunk 5]
```diff
@@ -1,8 +1,16 @@
 import styled from 'styled-components';
 import { ReactComponent as VideoIcon } from "./images/VideoIcon.svg";
 import { ReactComponent as SearchIcon } from "./images/SearchIcon.svg";
+import { NavLink } from 'react-router-dom';
 const bpMobile = ({ theme }) => theme.breakpoint.mobile;
 
+export const StyledNavLink = styled(NavLink)`
```
Comment (marcin10lw) [inline src/common/Header/styled.js:7]: Nice!

**Review by marcin10lw — CHANGES_REQUESTED** [review]
Nice you used react router v6!

File: src/App.js (hunk 1 above)
Comment (mikolajgraczyk) [reply src/App.js:11]: How about wrapping in index.js file?

Comment (mikolajgraczyk) [reply src/App.js:11]: How abou wrapping the app in index.js file?
obraz

**Pull request closed without merging**